src/effects/VST3/VST3UIValidator.cpp

```diff
@@ -163,6 +169,8 @@ void VST3UIValidator::OnClose()
    //Make sure that new state has been written to the caches...
    mAccess.Flush();
 
+   mWrapper.ParamChangedHandler = {};
```

Comment:
This line defends against a dangling pointer.  I would duplicate it also in the destructor of validator, just to be very certain it is never missed in exceptional situations.